PSL_catalog.json

```diff
@@ -113,22 +113,23 @@
     },
     "link_to_webapp": {
         "data": null,
-        "source": null,
+        "source": "https://compute.studio/PSLmodels/OG-USA/",
         "type": null,
         "start_header": null,
         "end_header": null
+        "data": "<a href=\"https://compute.studio/PSLmodels/OG-USA/\">Link to webapp</a>"
```

Comment:
@jdebacker . I looked at the source file for this line, and I can't figure out why the colon is highlighted in red. Any idea what is going on here?

Reply:
@rickecon  - missing comma.  Thanks for noticing.  Just fixed.